packages/form-footer/src/FormFooter.tsx

```diff
@@ -141,23 +192,41 @@ export default function FormFooter({
   ...rest
 }: FormFooterProps) {
   const { theme, darkMode } = useDarkMode(darkModeProp);
+
+  const backButton = {
+    text: backButtonProps?.text || backButtonText,
+    onClick: backButtonProps?.onClick || onBackClick,
+    leftGlyph:
+      backButtonProps?.leftGlyph === undefined ? (
+        <ArrowLeftIcon data-testid="lg-form_footer-back-button-icon" />
+      ) : (
+        backButtonProps.leftGlyph ?? undefined
+      ),
```

Comment:
API works like so:
- if `leftGlyph` is undefined, we default to the `ArrowLeftIcon`
- option to pass an explicit null to hide the icon
- option to pass a custom icon

open to feedback on this if any alternative preferences or if we have any established patterns we should adhere to